colosseum/validators.py

```diff
@@ -103,3 +105,31 @@ def is_color(value):
 
 
 is_color.description = '<color>'
+
+
+def is_shape(value):
+    """Check if given value is a shape."""
+    try:
+        # Remove extra outer  spaces
+        value = value.strip()
+
+        if value.startswith('rect'):
+            value = value.replace('rect(', '').replace(')', '')
```

Comment:
There's a few holes in the parsing scheme. If you don't have a closing brace, it will succeed; if you don't have an opening brace, it will fail (but only because the first value won't be parsed cleanly. I understand the desire to keep this parsing scheme simple (and not invoke regexes), but this needs to be a little more robust.

Reply:
Move to use a regex 🤷‍♂